### DISCUSSION TITLE
v1.65.0

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#3040] [Announcements]
## Highlights

Hello everyone, welcome to release `1.65.0` of Immich. We have the following news to share with you.

- Memories feature in the mobile app
- Add immich CLI inside the server container, so you can attach to the container and run the `immich upload ... --import` command directly for the import mechanism 
- More bugs fixed than you can find in your garden

Enjoy!

https://github.com/immich-app/immich/assets/27055614/d6db341f-0981-4664-8df1-a5290b07baad

## Support

- - - -

And as always, bugs are fixed, and many other improvements also come with this release.

Please consider supporting the project.

## Support

<p align="center">
<img src="https://media.giphy.com/media/LStqgGESXW8XnuCv5y/giphy.gif" width="250" title="SUPPORT THE PROJECT!"> 
</p>

If you find the project helpful, you can support Immich via the following channels.

- Monthly donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502)
- One-time donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502?frequency=one-time&sponsor=alextran1502)
- [Librepay](https://liberapay.com/alex.tran1502/)
- [buymeacoffee](https://www.buymeacoffee.com/altran1502)
- Bitcoin: 1FvEp6P6NM8EZEkpGUFAN2LqJ1gxusNxZX

It is a great way to let me know that you want me to continue developing and working on this project for years to come.

### Server
* feat(server): Add camera make and model to search criteria by @alextran1502 in https://github.com/immich-app/immich/pull/2982
* refactor(server): access permissions by @jrasm91 in https://github.com/immich-app/immich/pull/2910
* fix(server): add missing avi mime types and add tests by @uhthomas in https://github.com/immich-app/immich/pull/3001
* chore(server): check file extension for XMP instead of mimetype by @alex-phillips in https://github.com/immich-app/immich/pull/2990
* fix(server): empty tag responses should be considered valid by @mertalev in https://github.com/immich-app/immich/pull/2993
* chore(server): Image description disappears after toggle favorite by @KeszeiTheOne in https://github.com/immich-app/immich/pull/3009
* fix(server): live photos not playing in shared links/albums by @jrasm91 in https://github.com/immich-app/immich/pull/3008
* fix(server): use private cache by @uhthomas in https://github.com/immich-app/immich/pull/3017
* chore(server): add CLI tool to the server image by @ddshd in https://github.com/immich-app/immich/pull/2999

### Mobile
* fix(mobile): loading indicator position by @brighteyed in https://github.com/immich-app/immich/pull/2974
* chore(mobile): Capitalize Places cities in app by @alexleeelkins in https://github.com/immich-app/immich/pull/2985
* feat(mobile): memories by @alextran1502 in https://github.com/immich-app/immich/pull/2988
* fix(mobile): fix endless rendering of asset grid when scrolling by @fyfrey in https://github.com/immich-app/immich/pull/3010
* chore(mobile): minor UI tweak by @alextran1502 in https://github.com/immich-app/immich/pull/3021
* fix(mobile): set scrolling state only if changed by @fyfrey in https://github.com/immich-app/immich/pull/3034
* fix(mobile): rework album detail page header by @fyfrey in https://github.com/immich-app/immich/pull/3035
* chore(mobile): update flutter to 3.10.5 by @alextran1502 in https://github.com/immich-app/immich/pull/3036
* fix(mobile): avatar without last name by @alextran1502 in https://github.com/immich-app/immich/pull/3038
* feat(Android): find & delete corrupt asset backups by @fyfrey in https://github.com/immich-app/immich/pull/2963

### Web
* chore(web): Only show Copy button in HTTPS context by @alextran1502 in https://github.com/immich-app/immich/pull/2983
* fix(web): Add m: to search query upon loading results. by @aero31aero in https://github.com/immich-app/immich/pull/2954
* fix(web): aspect ratio for photos with Rotate 270 CW orientation by @brighteyed in https://github.com/immich-app/immich/pull/3003
* fix(web): persist info by @martabal in https://github.com/immich-app/immich/pull/3013
* fix(web): aspect ratio for videos with 90 CW and 270 CW orientation by @brighteyed in https://github.com/immich-app/immich/pull/3023
* fix(web): Share link multi-select download icon showing when not available #3006 by @faupau03 in https://github.com/immich-app/immich/pull/3027
* Fix(web): drag n drop shared link by @faupau03 in https://github.com/immich-app/immich/pull/3030

### Machine Learning
* fix(ml): setting different clip by @mertalev in https://github.com/immich-app/immich/pull/2987
* fix(ml): clear model cache on load error by @mertalev in https://github.com/immich-app/immich/pull/2951

### Other
* chore(ml): added testing and github workflow by @mertalev in https://github.com/immich-app/immich/pull/2969
* chore(deps): bump docker/setup-buildx-action from 2.7.0 to 2.8.0 by @dependabot in https://github.com/immich-app/immich/pull/3028

## New Contributors
* @alexleeelkins made their first contribution in https://github.com/immich-app/immich/pull/2985
* @aero31aero made their first contribution in https://github.com/immich-app/immich/pull/2954
* @ddshd made their first contribution in https://github.com/immich-app/immich/pull/2999

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.64.0...

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.65.0'>v1.65.0</a>.</em>

**Comment by charles-997** [comment]
> Add immich CLI inside the server container, so you can attach to the container and run the immich upload ... --import command directly for the import mechanism

When running immich CLI directly from the server container's CLI, should we still input the server URL?  Or do we just use something such as localhost instead?  I am looking to bypass the immich proxy container when using the CLI tool.

**Reply by ddshd** [reply]
You can use `http://immich_proxy:8080/api` with the default docker-compose

**Reply by charles-997** [reply]
Got it. Thanks for pointing me in the right direction.
